docs/packs/packs-format.md

```diff
@@ -49,6 +49,7 @@ The following fields are populated in the pack metadata:
 | currentVersion | String | The pack version, in the format of `x.x.x`. On the initial release this should be set to "1.0.0". See [here](#content-packs-versioning)|
 | author | String | The name of the organization (for partners) or developer (for individual contributions) which developed the integration. |
 | url | String | The URL to which users should refer to in case of support needed regarding the pack. Usually is the organization support URL or the developer GitHub repository. If left empty the default support site presented to users will be the [Live Community](https://live.paloaltonetworks.com/t5/cortex-xsoar-discussions/bd-p/Cortex_XSOAR_Discussions) site.|
+| videos | String | The Youtube video link of the pack.|
```

Comment:
Is that have to be a Youtube video?

Reply:
yes. Per the stressing timelines and capacity available from ENG for 6.8, merely a youtube embedded viewer was added.